bindings/rust/extended/s2n-tls-sys/templates/Cargo.template

```diff
@@ -36,8 +36,8 @@ stacktrace = []
 # unstable-foo = []
 
 [dependencies]
-aws-lc-rs = { version = "1" }
-libc = "0.2"
+aws-lc-rs = { version = "1.6.4" }
+libc = "0.2.121"
```

Comment:
libc 0.2.121 is needed because [aws-lc-sys versions before 0.24.0 required libc 0.2.121](https://github.com/aws/aws-lc-rs/blob/cdb1651d7398148688fa4c0106135b4a82ec1e49/aws-lc-sys/Cargo.toml#L69). 0.24.0 removed the libc dependency, so libc 0.2.62 should be acceptable for aws-lc-sys >= 0.24.0. However, I'm not aware of a way to specify different minimum versions depending on the version of a dependency in a build.